yoti/admin/views/yoti/tmpl/default.php

```diff
@@ -0,0 +1,139 @@
+                <div class="controls">
+                    <input type="text" name="yoti_app_id" id="yoti_app_id" placeholder="Yoti App ID"
+                           value="<?php if (!empty($this->data['yoti_app_id'])) echo htmlspecialchars($this->data['yoti_app_id']); ?>"
+                           class="form-control input-xlarge"
+                           required="true"
+                    />
+                    <span class="help-block"><?php echo JText::_("Copy the App ID from your Yoti App here"); ?></span>
+                </div>
+            </div>
+            <div class="control-group">
+                <label for="yoti_scenario_id" class="col-md-6 control-label"><?php echo JText::_("Scenario ID"); ?> <span><strong>*</strong></span></label>
+                <div class="controls">
+                    <input type="text" name="yoti_scenario_id" id="yoti_scenario_id" placeholder="Yoti Scenario ID"
+                           value="<?php if (!empty($this->data['yoti_scenario_id'])) echo htmlspecialchars($this->data['yoti_scenario_id']); ?>"
+                           class="form-control input-xlarge"
+                           required="true"
+                    />
+                    <span class="help-block">
+                        <?php echo JText::_("Scenario ID is used to render the inline QR code"); ?>
+                    </span>
+                </div>
+            </div>
+            <div class="control-group">
+                <label for="yoti_sdk_id" class="col-md-6 control-label">
+                    <?php echo JText::_("SDK ID"); ?> <span><strong>*</strong></span>
+                </label>
+                <div class="controls">
+                    <input type="text" name="yoti_sdk_id" id="yoti_sdk_id" placeholder="Yoti SDK ID"
+                           value="<?php if (!empty($this->data['yoti_sdk_id'])) echo htmlspecialchars($this->data['yoti_sdk_id']); ?>"
+                           class="form-control input-xlarge"
+                           required="true"
+                    />
+                    <span class="help-block"><?php echo JText::_("Copy the SDK ID from your Yoti App here"); ?></span>
+                </div>
+            </div>
+            <div class="control-group">
+                <label for="yoti_company_name" class="col-md-6 control-label"><?php echo JText::_("Company Name"); ?></label>
+                <div class="controls">
+                    <input type="text" name="yoti_company_name" id="yoti_company_name" placeholder="Company Name"
+                           value="<?php if (!empty($this->data['yoti_company_name'])) echo htmlspecialchars($this->data['yoti_company_name']); ?>"
+                           class="form-control input-xlarge"
+                    />
+                    <span class="help-block"><?php echo JText::_("To tailor our Yoti plugin please add your company name"); ?></span>
```

Comment:
Again, I feel like this is not explaining what`Comapany Name` does.

Reply:
This message was suggested by Kiran, so I will ask her what to change it to.

Reply:
"To tailor the login form messages please add your company name."